shuttle/schedules/__init__.py

```diff
@@ -31,3 +33,38 @@ def check_in(self):
     def logs(self):
         self.sc.check_roles_and_route(['Administrator'])
         return render_template('schedules/driver_logs.html')
+
+    @route('/users')
+    def users(self):
+        self.sc.check_roles_and_route(['Administrator'])
+        shuttle_user = db.get_users()
+
+        for key in shuttle_user:
+            shuttle_user[key]['name'] = db.username_search(shuttle_user[key]['username'])[0]['firstName'] + \
+                                        ' ' + db.username_search(shuttle_user[key]['username'])[0]['lastName']
+
+        return render_template('/schedules/users.html', **locals())
+
+    @route('load_user_data', methods=['POST', 'GET'])
+    def load_user_data(self):
+        self.sc.check_roles_and_route(['Administrator'])
+        username = json.loads(request.data).get('username')
+        return render_template('/schedules/user_modal.html', **locals())
+
+    @route('delete_user', methods=['POST'])
+    def delete_user(self):
+        self.sc.check_roles_and_route(['Administrator'])
+        username = json.loads(request.data).get('username')
+        print('deleting ' + username)
```

Comment:
remove test code

Reply:
> Could you add in a check that makes it so a user can't edit their own roles?

Added this functionality, and removed test code.